CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 Unreleased section should follow [Release Toolkit](https://github.com/newrelic/release-toolkit#render-markdown-and-update-markdown)
 ## Unreleased
 
+### enhancement
+- Remove dependence on amqp library for integration tests
+
```

Comment:
this is going to bump the minor version of the integration IIRC. I would not even add a changelog entry in this particular PR perhaps.

Reply:
Makes sense. Nothing changes in source code. Thanks.

Reply:
done.